### ISSUE TITLE
[sandstorm] Automated build not working when uploading to Sandstorm.io

### ISSUE THREAD

**Issue by engelgabriel** [RocketChat/Rocket.Chat#1496]
```sh
...** SANDSTORM SUPERVISOR: Starting up grain.
sandstorm/supervisor.c++:1144: warning: ip_tables kernel module not loaded; cannot set up transparent network forwarding.
*** Uncaught exception ***
sandstorm/sandstorm-http-bridge.c++:1762: failed: execvp(argvp[0], argvp): No such file or directory; argvp[0] = /opt/app/.sandstorm/launcher.sh
stack: 0x5057b9 0x5057aa 0x472b91
** SANDSTORM SUPERVISOR: Grain still in use; staying up for now.
** SANDSTORM SUPERVISOR: Grain still in use; staying up for now.
** SANDSTORM SUPERVISOR: Grain still in use; staying up for now.
** SANDSTORM SUPERVISOR: Grain still in use; staying up for now.
** SANDSTORM SUPERVISOR: Grain still in use; staying up for now.
** SANDSTORM SUPERVISOR: Grain still in use; staying up for now.
** SANDSTORM SUPERVISOR: Grain no longer in use; shutting down.
** SANDSTORM SUPERVISOR: Starting up grain.
sandstorm/supervisor.c++:1144: warning: ip_tables kernel module not loaded; cannot set up transparent network forwarding.
*** Uncaught exception ***
sandstorm/sandstorm-http-bridge.c++:1762: failed: execvp(argvp[0], argvp): No such file or directory; argvp[0] = /opt/app/.sandstorm/launcher.sh
stack: 0x5057b9 0x5057aa 0x472b91
** SANDSTORM SUPERVISOR: Grain still in use; staying up for now.
** SANDSTORM SUPERVISOR: Grain no longer in use; shutting down.
** SANDSTORM SUPERVISOR: Starting up grain.
sandstorm/supervisor.c++:1144: warning: ip_tables kernel module not loaded; cannot set up transparent network forwarding.
*** Uncaught exception ***
sandstorm/sandstorm-http-bridge.c++:1762: failed: execvp(argvp[0], argvp): No such file or directory; argvp[0] = /opt/app/.sandstorm/launcher.sh
stack: 0x5057b9 0x5057aa 0x472b91
```

**Comment by engelgabriel** [comment]
I am not the only one: https://github.com/RocketChat/Rocket.Chat.Cordova/issues/26

**Comment by Sing-Li** [comment]
Chroot `/start.js` is missing in the travis built SPK, vs the one at the app store.  @jparyani  - can you pls shed some light on this?

**Comment by pdehaye** [comment]
(this is a question)
Who packages the sandstorm Rocket Chat app? It is officially signed by @engelgabriel , but where is the script that does the packaging?

**Comment by engelgabriel** [comment]
I did the only official packaging to date... the key is on my machine. But what we are trying above was to use the unsigned package from https://github.com/RocketChat/Rocket.Chat/blob/develop/.travis.yml

**Comment by Sing-Li** [comment]
Fixed with #1526
